cmd/storagenode-updater/main.go

```diff
@@ -212,6 +212,8 @@ func update(ctx context.Context, nodeID storj.NodeID) (err error) {
 		} else {
 			log.Printf("%s version is up to date\n", runCfg.ServiceName)
 		}
+	} else {
+		log.Printf("%s version is up to date\n", runCfg.ServiceName)
```

Comment:
maybe just do `return nil` below `// TODO remove old binary ??` and do `log.Printf("%s version is up to date\n", runCfg.ServiceName)` outside if statement? to have only one `Printf`